sdks/cpp/include/agones/sdk.h

```diff
@@ -39,6 +39,13 @@ class SDK {
   // Marks the Game Server as ready to receive connections
   AGONES_EXPORT grpc::Status Ready();
 
+  // Self marks this gameserver as Allocated.
```

Comment:
What is "Self"?

Reply:
"self marks" is a phrase -- basically instead of having the gameserver be allocated through the allocation service, the gameserver does the allocation itself. The comment matches the [comment in the go sdk](https://github.com/googleforgames/agones/blob/master/sdks/go/sdk.go#L69) (as do all of the other comments in this file).